docs/root/configuration/http/http_filters/rate_limit_quota_filter.rst

```diff
@@ -0,0 +1,180 @@
+    on_no_match:
+      action:
+        name: default-bucket
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaBucketSettings
+          bucket_id_builder:
+            bucket_id_builder:
+              "name":
+                string_value: "default-rate-limit-quota"
+          reporting_interval: 60s
+          deny_response_settings:
+            http_status_code: 429
+          no_assignment_behavior:
+            blanket_rule: ALLOW_ALL
+          expired_assignment_behavior:
+            fallback_rate_limit:
+              requests_per_time_unit:
+                requests_per_time_unit: 1000
+                time_unit: 1s
+
+
+Rate Limit Quota Override
+-------------------------
+
+Rate limit filter :ref:`configuration <envoy_v3_api_msg_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaFilterConfig>` can be overridden
+at the virtual host or route levels using the :ref:`RateLimitQuotaOverride <envoy_v3_api_msg_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaOverride>`
+configuration. The more specific configuration fully overrides less specific configuration.
+
+Matcher extensions
+------------------
+
+TODO
+
+Statistics
+----------
+
+The rate limit filter outputs statistics in the *cluster.<route target cluster>.rate_limit_quota.* namespace.
+429 responses or the configured
+:ref:`rate limited status <envoy_v3_api_field_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaBucketSettings.DenyResponseSettings.http_status>`
+are emitted to the normal cluster :ref:`dynamic HTTP statistics <config_cluster_manager_cluster_stats_dynamic_http>`.
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  buckets, Counter, Total number of request buckets created
+  assignments, Counter, Total rate limit assignments received from the rate limit quota service
+  error, Counter, Total errors contacting the rate limit quota service
+  over_limit, Counter, Total requests that exceeded assigned rate limit
+  no_assigment, Counter, "Total requests that were applied the
+  :ref:`no_assigment_behavior <envoy_v3_api_field_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaBucketSettings.no_assignment_behavior>`"
+  expired_assigment, Counter, "Total requests that were applied the
+  :ref:`expired_assignment_behavior <envoy_v3_api_field_extensions.filters.http.rate_limit_quota.v3.RateLimitQuotaBucketSettings.expired_assignment_behavior>`"
+
+Dynamic Metadata
+----------------
+
+TODO
+
+Runtime
+-------
+
+The HTTP rate limit quota filter supports the following runtime settings:
+
+rate_limit_quota.http_filter_enabled
+  % of requests that will call the rate limit service. Defaults to 100.
+
+rate_limit_quota.http_filter_enforcing
+  % of requests that that will have the rate limit service decision enforced. Defaults to 100.
+  This can be used to test what would happen before fully enforcing the outcome.
```

Comment:
In general we don't hard code runtime settings anymore. Can you add runtime specifically to the config if you want it? This allows customizing the path, using different paths for different filters, etc.

Reply:
FYI @yanavlasov

Reply:
Done